### PR TITLE
ahaha i swear im still working on this ahaha

### PR DESCRIPTION
**Pull request by VKong6019**
i uh am almost done with the brute shuffle _algorithm_ :D

~~i just need to test this and refactor now hehe~~

https://github.com/sandboxnu/boba-buddies/issues/30#issue-1388760797

### CODE REVIEW

File: utils/generatePairs.ts [hunk 1]
```diff
@@ -1,34 +1,51 @@
...
 import { UsersDataManager } from "../database/usersDataManager";
+import { PoolDataManager } from "../database/poolDataManager";
 
 //TODO: change this to actual channel id when deploying
 const TESTING_CHANNEL_ID = "C02J6R0SUSX";
 export const BOT_USER_ID = "U02J904RH1S";
 const userManager = new UsersDataManager();
+const poolManager = new PoolDataManager();
 
 export const generatePairs = async (app: App) => {
```
Comment (VKong6019) [inline utils/generatePairs.ts:11]: I haven't tested this particular functionality yet, but will soon and the `poolDataManger` is already tested.

**Comment by VKong6019** [comment]
there's so much spaghetti everywhere 😢

**Comment by jia614** [comment]
omo project mommy ❤️

File: database/poolDataManager.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+			}
+		return await DB_CLIENT.delete(delParams).promise();
+	}
+
+	async popPair(user: string): Promise<Map<string, string>> {
+		const userPool = await this.getPoolForUser(user);
+		const randomPair = userPool[Math.floor(Math.random() * userPool.length)];
+		await this.deletePair(new Map(Object.entries(randomPair)));
+
+		return randomPair;
```
Comment (VKong6019) [inline database/poolDataManager.ts:56]: omfg im dumb and i didn't account for previously popped users. this a bug

Comment (VKong6019) [reply database/poolDataManager.ts:56]: planning to do: configure this method to take in a list of already paired users and run a filter query into dynamodb

**Review by jia614 — APPROVED** [review]
lgtm 👍 😄

**Pull request merged**